quiz.js

```diff
@@ -37,9 +50,23 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+  name: 'Stark' }, { $push: {person: { name: 'Ned'}}});
 
+db.houses.update({
+  name: 'Stark' }, { $push: {person: { name: 'Arya'}}});
 
+db.houses.update({
+  name: 'Stark' }, { $push: {person: { name: 'Sansa'}}});
 
+db.houses.update({
+  name: 'Targaryen' }, { $push: {person: {name: 'Viserys'}}});
+
+db.houses.update({
+  name: 'Targaryen' }, { $push: {person: {name: 'Daenerys'}}});
+
+db.houses.update({
+  name: 'Arryn' }, { $push: {person: {name: 'Jon'}}});
 
 // your code ends here
```

Comment:
All correct! This could get a little tedious, though, if we needed to enter a large number of people. A slightly DRYer option might be to use Mongo's [$each](http://docs.mongodb.org/manual/reference/operator/update/each/) operator, like this : 
```
db.houses.update( { name : 'House Stark' },  { $push : { people : { $each : 
  [
    'Ned Stark', 
    'Arya Stark', 
    'Sansa Stark'
  ]
} } } );
```